seqr/views/utils/pedigree_info_utils.py

```diff
@@ -215,6 +228,8 @@ def _parse_header_columns(header, allow_id_update):
         elif 'indiv' in key and 'previous' in key:
             if allow_id_update:
                 column = JsonConstants.PREVIOUS_INDIVIDUAL_ID_COLUMN
+        elif update_features and 'hpo' in key and 'term' in key:
```

Comment:
why don't we enforce a strict schema for the headers in this file like checking if key == 'HPO Terms'?

Reply:
Because we wanted to make it easier for people to generate these files on their own, and being flexible in what the column names  are helps avoid nitpicky errors that are hard for non-technical users to manage. For things like RNA data uploads or other data genrated programmatically we are very strict about column formatting, but for use cases like these where an average user is manually making an excel file we don;t want to trip them up because we expect "HPO Terms" and they put "Hpo term"

Reply:
this makes sense. I guess one of the only downsides is that being flexible in the code means it's a little harder for the reader to figure out what is allowed, but that's solved by examples in the unit tests